packages/hop-node/src/config/validation.ts

```diff
@@ -68,32 +69,76 @@ export async function validateConfig (config?: FileConfig) {
     Chain.Polygon
   ]
 
+  const validTokenKeys = [
+    'USDC',
+    'USDT',
+    'DAI',
+    'ETH',
+    'MATIC',
+    'WBTC'
+  ]
+
+  let enabledChains: string[] = []
   if (config.chains) {
-    const networkKeys = Object.keys(config.chains)
-    await validateKeys(validNetworkKeys, networkKeys)
+    enabledChains = Object.keys(config.chains)
+    validateKeys(validNetworkKeys, enabledChains)
+    const validChainKeys = ['rpcUrl', 'waitConfirmations']
+    for (const chain in config.chains) {
+      validateKeys(validChainKeys, Object.keys(config.chains[chain]))
+      if (!config.chains[chain]) {
+        throw new Error(`RPC config for chain "${chain}" is required`)
+      }
+      const { rpcUrl, waitConfirmations } = config.chains[chain]
+      if (!rpcUrl) {
+        throw new Error(`RPC url for chain "${chain}" is required`)
```

Comment:
I believe all bonders, regardless of their path, need access to a mainnet RPC endpoint in order to sync `TransferRootSet` (and others, if it is an ORU). I believe we should enforce this here. Maybe with `enabledChains.push('mainnet')` on line 84?

Reply:
nice catch. The validation now requires that `ethereum` chain config is set. Resolved with https://github.com/hop-protocol/hop/pull/346/commits/1878d754ab9e40806a9a1748530403686306e2bb